backend/api/models/Location.py

```diff
@@ -0,0 +1,16 @@
+from mongoengine.fields import (

```

Comment:
If you're importing one thing you don't need the parantheses
```suggestion
from mongoengine.fields import StringField
```